src/store/contacts.js

```diff
@@ -71,8 +75,31 @@ const mutations = {
 	},
 }
 
-const getters = {}
+const getters = {
+	/**
+	 * Gets a contact's avatar
+	 *
+	 * @param {Object} state the store data
+	 * @returns {function({String}): {String}}
+	 */
+	getAvatarForContact: (state) => (uri) => {
+		const contact = state.contactByEMail[stripMailTo(uri)]
+		return contact?.isUser ? undefined : contact?.avatar
+	},
+
+	isContactAnUser: (state) => (uri) => {
```

Comment:
```suggestion
	isContactAUser: (state) => (uri) => {
```
https://english.stackexchange.com/questions/105116/is-it-a-user-or-an-user

Reply:
God-dammit English